recipe/meta.yaml

```diff
@@ -96,19 +99,30 @@ test:
     {% endfor %}
 
     # Test ncurses library arguments.
-    - pkg-config ncursesw --libs | grep "^-L${PREFIX}/lib -lncursesw -ltinfow$"
-    - pkg-config tinfow --libs | grep "^-L${PREFIX}/lib -ltinfow$"
-    - pkg-config ncurses --libs | grep "^-L${PREFIX}/lib -lncurses -ltinfo$"
-    - pkg-config tinfo --libs | grep "^-L${PREFIX}/lib -ltinfo$"
+    - pkg-config ncursesw --libs | grep "^-L${PREFIX}/lib .* -lncursesw$"
+    - pkg-config tinfow --libs | grep "^-L${PREFIX}/lib .* -ltinfow$"
+    - pkg-config ncurses --libs | grep "^-L${PREFIX}/lib .* -lncursesw$"
+    - pkg-config tinfo --libs | grep "^-L${PREFIX}/lib .* -ltinfow$"
 
     # Usage tests
     - bash run_test.sh    # [osx]
-    - python run_test.py  # [osx]
+    # Python test will not pass until it is recompiled against ncurses 6.0
+    #- python run_test.py  # [osx]
```

Comment:
We should just move this over to the `python` feedstock. It's a poor separation of concerns for us to test this here.